config/routes.rb

```diff
@@ -8,7 +8,8 @@
   post '/login',    to: "sessions#create"
   delete 'logout',  to: "sessions#destroy"
 
-  resources :users,   except: :new
+  resources :users,         except: :new
+  resources :categories,    except: [:new, :create, :edit, :update, :destroy]
```

Comment:
@shokogyu You can use `only`, instead of `except`. So you can choose what to include.